src/TimeGridEvent.js

```diff
@@ -42,10 +46,10 @@ function TimeGridEvent(props) {
         onDoubleClick={onDoubleClick}
         style={{
           ...userProps.style,
-          top: `${top}%`,
-          height: `${height}%`,
-          [rtl ? 'right' : 'left']: `${Math.max(0, xOffset)}%`,
-          width: `${width}%`,
+          top: stringifyPercent(top),
+          [rtl ? 'right' : 'left']: stringifyPercent(xOffset),
+          width: stringifyPercent(width),
+          height: stringifyPercent(height),
```

Comment:
Sometimes the algorithm needs `calc()` so I just made it as `'string' || 'number'`. number will be followed by `%`.

Maybe you think it is no good that I didn't make it all `string` only. Frankly I tried it but you know that plain css string is quite annoying for testing, right? 

![image](https://user-images.githubusercontent.com/12825679/66562846-73fc1480-eb97-11e9-8824-e9c327d62f3e.png)

I also tried to `parse()` and `floor()` but... yeah... too much. so this is why.